src/App.tsx

```diff
@@ -1,61 +1,123 @@
 import './App.scss';
+import { FC, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList/TodoList';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
+// import { UserInfo } from './components/UserInfo/UserInfo';
+// import { TodoInfo } from './components/TodoInfo/TodoInfo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: FC = () => {
+  const [todoTitle, setTodoTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [visibleTodos, setVisibleTodos] = useState(todos);
```

Comment:
visibleTodos  do not very match in this case as for me

Reply:
in case we need to filter smth